src/proto/Aion.proto

```diff
@@ -25,6 +25,9 @@ message SigningInput {
 
     // Private key.
     bytes private_key = 7;
+
+    // Timestamp (256-bit number)
+    bytes timestamp = 8;
```

Comment:
Isn't UInt64 enough?

Reply:
This is UIn64 max value `18446744073709551615`. If we use unixepoch in nanoseconds:
<img width="658" alt="Screen Shot 2020-08-23 at 21 43 31" src="https://user-images.githubusercontent.com/1442927/90993052-b947eb80-e589-11ea-9f64-0421510febc5.png">

Reply:
Good point, let me change it